q_it_argument_error.rb

```diff
@@ -0,0 +1,8 @@
+class QItArgumentError < StandardError
```

Comment:
You may be able to do this
`class QItArrgumentError < StandardError; end`

Reply:
yes that works. But wanted to communicate the object that gives the error. So added the `initialize` method.